### PR TITLE
#4 Project about cloning Apple Website using Backgrounds and Gradients

### PR DESCRIPTION
**Pull request by guillainbisimwa**
This all the changes we made to implement this project: 
- Fix errors from w3c validations
- Formating CSS code
- Added IMG vertical alignment fixing
- Updated search input bar style on the main menu
- Styling the menu
- Working on menu style
- Aligned the footer nav with the other footer elements
- General Layout Styling, added images, spacing and styles to the whole page
- Create the apple top menu
- Added menu background and bg image for main

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Apple</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <!-- Main content -->
+    <main>
```
Comment (kevcap) [inline index.html:12]: - [x] [OPTIONAL] Consider remove that obvious comments;

File: index.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            </div>
+            <div id="ad4" class="img-ads cpointer">
+                
+            </div>
+        </div>
+
+    </main>
+
+    <!-- Footer  -->
+    <footer class="d-flex">
```
Comment (kevcap) [inline index.html:59]: - [x] [OPTIONAL] Consider remove that obvious comments;

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️


#### Hey team @guillainbisimwa and @chubaquelo (Hi again!), Is a pleasure to review such a good project! Congrats for the amazing work you guys already did, the project looks really nice and I can see that you guys are working hard on it, and we're almost done, there are just some minor changes that you guys have to implement to proceed:


_Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, in order to get more knowledge and have a better project to add at your portfolio.

## [REQUIRED CHANGES]
#### Good practices REQUIREMENTS (Those changes to make you code like a pro 👽)⤵
_**You guys rocked with the good practices!**_ 🥇
#### Clone REQUIREMENTS 👥 (Those adjusts to make your page really look like the real one)⤵
- [x] Hey team, this clone should contains four elements which are links (anchor tags) (to other articles or #), looks like you guys used divs with a cursor: pointer rule, kindly fix that and make it actual anchor tags 😎 ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))


## [OPTIONAL CHANGES]
- [x] Add a favicon to make it more similar to the original page;
- [x] Put your `css` files in a css folder. (To make your repo more professional)
- [x] Remove obvious comments. For example: 
image
and:
image



### Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀

👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question about this review? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi Team! 👋 
### Great job implementing the requested changes!! 🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****

 Good Luck and Happy Coding! 🍀 👩🏾‍💻

**Pull request merged**